src/epython/environment.py

```diff
@@ -11,60 +11,33 @@
 Date:
```

Comment:
I wanted this file to be any environmental abstractions that would let the user override the behavior or epython.